packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.stories.ts

```diff
@@ -126,6 +134,23 @@ To improve the ability for users to visually scan values, applications should se
 </details>
 `;
 
+const unitDescription = `A unit for the column may be configured by providing a \`nimble-unit-<name>\` element as content (in addition to the column label). Unit elements represent a set of related, scaled units, e.g. \`nimble-unit-byte\` represents bytes, KB, MB, etc. Values are converted from a source unit (e.g. bytes) to the largest scaled unit (e.g. KB, MB, etc.) that can represent that value with magnitude >= 1. The source data for the column is expected to be given in the base unit specified in the tag name, e.g. for \`nimble-unit-byte\`, a source value should be a number of bytes.
+
+<details>
```

Comment:
I think I might advocate for us to move the unit docs to a separate section in an mdx file for this column and/or a separate story. As we add units it'll eventually be a lot of info to cram into a details section of a parameter description.

But that doesn't need to happen in this PR, it can happen as we add more units. I'm just throwing it out now for feedback / awareness.

Reply:
Agreed. I think most of the options in the table have too many details inline that are worth having but should be moved to discussion in the MDX. I don't think we need to track that explicitly but likely would predicate adding additional units after this PR on adding an mdx and cleaning up units at least.